### ISSUE TITLE
`hwdb` build breaks when creating a VM on `release-18.09`

### ISSUE THREAD

**Issue by Ma27** [NixOS/nixpkgs#55238]
## Issue description

Creating NixOS VMs using `nixos-build-vms` currently fails on `release-18.09`:

```
copying path '/nix/store/7gbd1as2whimg6a0d6rfdis5c9syxsl2-linux-4.14.97' from 'https://cache.nixos.org'...
building '/nix/store/y32jwj5k1bw6ndsmqx6p7zh9vsf72fjm-etc-hostname.drv'...
building '/nix/store/163s3gx6dkarivj92xgapr0qsl4ra595-etc-hosts.drv'...
building '/nix/store/7fmx7qvm7ws2i9y8cxyk97vx24005a88-etc-os-release.drv'...
building '/nix/store/m9j0nbj1qpwsh5lkmggpppnvki65akrr-hwdb.bin.drv'...
building '/nix/store/4n1qczgmwf6f9875fyvb3v81jhys3ynx-issue.drv'...
building '/nix/store/k1rwwiad9w5gnx21y3si0z763gzvd4sl-nixos-version.drv'...
Adding hwdb files for package /nix/store/v7lqcna63kxi5s520r931py52nlgy0ac-systemd-239.20190110
building '/nix/store/7snm63hayhhq4cyi8ncjk8abs28a7nan-pam-environment.drv'...
building '/nix/store/bb3486g1xhdqp83bxrg1dyajlz5lxg30-reload-container.drv'...
Adding hwdb files for package /nix/store/7wmwwpzxcwggp1yrqamcwc5b5w51n7vf-mdadm-4.0
Adding hwdb files for package /nix/store/siyf0iar7xxn8nissk0w89n4jigbgk4a-lvm2-2.02.177
Adding hwdb files for package /nix/store/yn2ywcllc9v861583r8rdkd9xvqz9d14-bcache-tools-1.0.7
Adding hwdb files for package /nix/store/gnk5gc6mm6jjrr9ix5c10rjdardmsyf3-udisks-2.8.1
Adding hwdb files for package /nix/store/ljhs6h0jfnggnksl846axfq0zsi54bz4-extra-udev-rules
Adding hwdb files for package /nix/store/mk1bfg6ja2s2k3gvnn1c4g2dq1nrgg06-extra-hwdb-file
Generating hwdb database...
builder for '/nix/store/m9j0nbj1qpwsh5lkmggpppnvki65akrr-hwdb.bin.drv' failed with exit code 127
cannot build derivation '/nix/store/ndzw8wahn6a7fi75b5bsfgfhbl9fiyar-etc.drv': 1 dependencies couldn't be built
```

### Steps to reproduce

* Create an empty VM expression:

  ```nix
  { # vm.nix
    test = {};
  }
  ```

* Run `NIX_PATH=nixpkgs=/path/to/nixpkgs/with/18.09 nixos-build-vms vm.nix`

## Technical details

Tested with a local checkout of `nixpkgs` at `release-18.09` (currently at 38a99219f3cbf45d6a2c8b6d58cd4eaf9ce1af13) on a `x86_64-linux` with Nix 2.1.3.

**Comment by fpletz** [comment]
Works for me on d770864744167bceb4e92c6d6f1da7e5e99a6ef3.

**Comment by Ma27** [comment]
I'm admittedly not sure what the original cause for this was, but the problem appears to be fixed now on `release-18.09`, so let's close this.
